change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW

```diff
@@ -386,6 +399,9 @@ PROCEDURE NAMEFMBYLEVEL
        ERROR503CHR=FORMAT("Banno Chg Addr Err - Nm LOC #####9",ACCTLVLLOCS(TMPLOOP))+" FM-"+FMERROR
        PARSEDTEXT(1)=FORMAT("Banno Chg Addr Err - Nm LOC #####9",ACCTLVLLOCS(TMPLOOP))
        PARSEDTEXT(2)="FM-"+FMERROR
+       IF ALLOWINMEMOMODE=TRUE AND SYSMEMOMODE=TRUE AND
```

Comment:
If the intent is just to record FM failure while the system is in memo mode, I would suggest we just check SYSMEMOMODE value. No need to check the other variables.